docs/src/further_topics/ugrid/other_meshes.rst

```diff
@@ -0,0 +1,223 @@
+.. _other_meshes:
+
+Converting Other Mesh Formats
+*****************************
+
+Iris' Mesh Data Model is based primarily on the CF-UGRID conventions  (see
+:doc:`data_model`), but other mesh formats can be converted to fit into this
+model, **enabling use of Iris' specialised mesh support**. Below are some
+examples demonstrating how this works for various mesh formats.
+
+.. contents::
+    :local:
+
+`FESOM 1.4`_ Voronoi Polygons
+-----------------------------
+..  figure:: images/fesom_mesh.png
+    :width: 300
+    :alt: Sample of FESOM mesh voronoi polygons, with variable numbers of sides.
+
+A FESOM mesh encoded in a NetCDF file includes:
+
+* X+Y point coordinates
+* X+Y corners coordinates of the Voronoi Polygons around these points -
+  represented as the bounds of the coordinates
+
+To represent the Voronoi Polygons as faces, the corner coordinates will be used
+as the **nodes** when creating the Iris
+:class:`~iris.experimental.ugrid.mesh.Mesh`.
+
+.. dropdown:: :opticon:`code`
+
+    .. code-block:: python
+
+        >>> import iris
+        >>> from iris.experimental.ugrid import Mesh
+
+
+        >>> temperature_cube = iris.load_cube("my_file.nc", "sea_surface_temperature")
+        >>> print(temperature_cube)
+        sea_surface_temperature / (degC)    (time: 12; -- : 126859)
+            Dimension coordinates:
+                time                             x        -
+            Auxiliary coordinates:
+                latitude                         -        x
+                longitude                        -        x
+            Cell methods:
+                mean where sea              area
+                mean                        time
+            Attributes:
+                grid                        'FESOM 1.4 (unstructured grid in the horizontal with 126859 wet nodes;...
+                ...
+
+        >>> print(temperature_cube.coord("longitude"))
+        AuxCoord :  longitude / (degrees)
+            points: <lazy>
+            bounds: <lazy>
+            shape: (126859,)  bounds(126859, 18)
+            dtype: float64
+            standard_name: 'longitude'
+            var_name: 'lon'
+
+        # Replace existing face AuxCoords with MeshCoords.
```

Comment:
```suggestion
        # Replace existing face AuxCoords with new MeshCoords.
```

Also I think this step may be suprising to a new user, as in why do we have to remove the existing AuxCoords. It isn't immediately obvious why you can't have both. Maybe adding something like "to avoid coord name collision"?

Reply:
08b58740da3b91129d6d32225dd4273dd54de3a0